html/about.html

```diff
@@ -0,0 +1,137 @@
+                    <li><a href="#news">News</a></li>
+                </ul>
+                <a class="campagn" href="#contact">CC Campagn</a>
+            </div>
+        </nav>
+    </header>
+
+    <section class="hero">
+        <div class="evt-title">
+            <h3>"Hello! Sharing world"</h3>
+            <h1>WORLD GREATEST EVENT ON NEW TECHS</h1>
+        </div>
+
+        <div class="evt-desc">
+            <p>The Global Summit brings together the community of experts, academics, and activists who comprise the
+                Creative Commons affiliale network in a different country every two. This year, we hope to expand our
+                invitation list including organizations and individuals</p>
+        </div>
+
+        <div class="evt-d-p">
+            <h2>Please contact us per Email for any further questions about CC Global Summit 2015!</h2>
+            <h5>ccsumit2015@cckorea.org</h5>
+        </div>
+    </section>
+
+    <section class="past-evnt">
+        <div class="container">
+
+            <h2>Creative Commons Global Summit 2015 Logo</h2>
+            <div class="line"></div>
+            <span></span>
+            <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July.</p>
+
+        </div>
+
+        <div class="ctn-logo">
+            <p>Tech<span>Meet</span></p>
+        </div>
+        <div class="ctn-past-evnt">
+            <h3>See the past CC Global Summits</h3>
+            <span></span>
+            <p>Take a look at the last two CC Global Summits who took place in Buenos Aires and in Warsaw</p>
+
+            <div class="frame-ctn">
+                <div class="frame">
+                    <div class="ctn-frame">
+                        <img src="../images/cc_1.jpg" alt="event image">
+                    </div>
+                    <div class="img-overlay">
+                        <h3>2013</h3>
+                        <p>CC Global Summit 2013 Buenos Aires</p>
+                    </div>
+                </div>
+                <div class="frame">
+                    <div class="ctn-frame">
+                        <img src="../images/cc_2.jpg" alt="event image">
+                    </div>
+                    <div class="img-overlay">
+                        <h3>2013</h3>
+                        <p>CC Global Summit 2013 Buenos Aires</p>
+                    </div>
+                </div>
+            </div>
+            
+        </div>
+    </section>
+
+    <section class="ctn-partner">
+        <h3>Partner</h3>
+        <div class="line"></div>
+
+        <div class="partner">
+            <img class="ptn-img" src="../images/p_1.png" alt="">
+            <img class="ptn-img" src="../images/p_2.png" alt="">
+            <img class="ptn-img" src="../images/p_3.png" alt="">
+            <img class="ptn-img" src="../images/p_4.png" alt="">
+            <img class="ptn-img" src="../images/p_5.png" alt="">
+        </div>
+    </section>
```

Comment:
- [ ] Kindly move this section to the index.html page so it looks like the Behance design👍🏿